src/app/subscribe/components/Subscription/index.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { useState } from 'react';
+import { Button } from '@/components/Common';
+import { cn } from '@/utils';
+import { AsyncBoundaryWithQuery } from '@/react-utils';
+import { FollowListFetcher, FollowerListFetcher } from '../SubscribeFetcher';
+import SubscriptionFallback from '../SubscriptionFallback';
+import FollowListContent from './FollowListContent';
+import FollowerListContent from './FollowerListContent';
+
+export default function Subscription() {
+  const [selectedButton, setSelectedButton] = useState<'내가구독' | '나를구독'>(
+    '내가구독',
+  );
+  const [currentPage, setCurrentPage] = useState<number>(0);
+
+  return (
+    <section className="fixed px-20">
+      <div className="text-bold text-[17px] text-primary font-luckiest px-30 pt-100">
+        SUBSCRIPTION
+      </div>
+      <div className="flex flex-row gap-5 justify-center pb-30">
+        <Button
+          className={cn(
+            'border',
+            selectedButton === '내가구독' && 'border-primary',
+            'bg-white rounded-30 p-6 text-[12px] transition-all duration-100',
```

Comment:
조건문이 없는 경우 문자열의 경우엔 합쳐주세요.
```suggestion
            selectedButton === '내가구독' && 'border-primary',
            'border bg-white rounded-30 p-6 text-[12px] transition-all duration-100',
```